Dockerfile

```diff
@@ -18,14 +18,15 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		python3-jinja2 \
 		python3-mysqldb \
 		python3-requests \
+        python3-yaml \
```

Comment:
Could you indent this line the same as the other dependencies? It looks really sloppy.

Reply:
I see that you did 👍